_posts/2020-08-31-JabRef5-1..md

```diff
@@ -0,0 +1,40 @@
+---
+title:  JabRef 5.1 Release 
+id: jabref-5-1
+author: "[JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS)" 
+---
+
+We are proud to announce the release the next milestone of JabRef. Today we are releasing Jabref 5.1.
+
+## Notable Changes and New Features
+
+After the release of the previous major milestone 5.0 we intensively focused on performance and bug fixes. We managed to speed up JabRef 5.1 significantly, especially when used with many groups and/or larger libraries.
+
+We didn't just focus on performance and bug fixes, but also on a couple of new features:
+
+For users using our integrated fetchers we added the option to automatically download the associated fulltext (if available) on import. Related to that is the new support of [mEDRA DOIs](https://www.medra.org/) to retrieve metadata. 
+We improved the integrity check which can now detect duplicated DOIs. This should be a reliable way to check for duplicates in the library.
+For LaTeX users, JaRef now supports the entry types and fields of [`biblatex-software`](https://ctan.org/pkg/biblatex-software) out of the box.
+
+Additionally to these improvements under the hood, we also tweaked the user interface:
+
+We added tooltips for nearly all fields in the [entry editor](https://docs.jabref.org/advanced/entryeditor) to show a description of the fields purpose. The comment field especially now supports markdown.
+A progress indicator in the toolbar provides an overview of running background tasks and shows their average progress. Also the toolbar should now scale more nicely in smaller windows.
+The preferences dialog got some smaller visual improvements and the structure was a bit refined.
+A major change affects users of Mac computers running Mac OS Catalina. You will no longer get any warning about insecure software. JabRef for Mac is now completely signed and certified. We also took the opportunity to visually adjust the installer.
+
+![JabRef 5.1 DMG screen](/img/jabref-5-1-macDmg.png)
```

Comment:
I don't think that this update is important enough to be the only picture in the release notes. What about the download on import and the progress indicator instead?

Reply:
That was my suggestion as well. We should add some more screenshots and maybe also from the preferennces

Reply:
We have the markdown file now. Is it OK to have both or should we remove this one?